pkg/attestation/crafter/crafter.go

```diff
@@ -418,6 +422,9 @@ func (c *Crafter) ResolveEnvVars(ctx context.Context, attestationID string) erro
 		}
 	}
 
+	// Resolve runner information
+	c.ResolveRunnerInfo(ctx)
```

Comment:
I do not think you need to expose this method anymore right?

Reply:
Good point, thanks, done.